apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/SchemaTypeSpecBuilder.kt

```diff
@@ -134,7 +134,7 @@ class SchemaTypeSpecBuilder(
 
   private fun nestedTypeSpecs(): List<Pair<String, TypeSpec>> {
     return fields.filter(Field::isNonScalar).map {
-      it.formatClassName() to it.toTypeSpec(
+      it.formatClassName(context) to it.toTypeSpec(
```

Comment:
It feels a bit heavy handed to pass `context` around when all that's needed is `singularizeTypes` but I'm no expert of these classes so works for me as is too :)

Reply:
Updated to only pass in the boolean.